src/transformers/models/llama/tokenization_llama.py

```diff
@@ -245,7 +250,11 @@ def tokenize(self, text: "TextInput", add_special_tokens=False, **kwargs) -> Lis
         if self.legacy or len(text) == 0:
             return super().tokenize(text, **kwargs)
 
-        tokens = super().tokenize(SPIECE_UNDERLINE + text.replace(SPIECE_UNDERLINE, " "), **kwargs)
+        text = text.replace(SPIECE_UNDERLINE, " ")
+        if kwargs.pop("add_prefix_space", self.add_prefix_space):
+            text = SPIECE_UNDERLINE + text
```

Comment:
do you want to do these additions before you do the replace?

Reply:
doesn't make a different I think